src/customer/components/CustomerPart.js

```diff
@@ -125,7 +158,13 @@ const CustomerPart = () => {
               <DetailRow title="Alloy" value={alloy}></DetailRow>
             </CardContent>
           </Grid>
-          <Grid container item xs={3} direction="column" justify="space-between">
+          <Grid
+            container
+            item
+            xs={3}
+            direction="column"
+            justify="space-between"
+          >
             <Typography variant="h6">£1,200</Typography>
```

Comment:
£ should be &#163;

Reply:
It's  a minor content change so I'm going to leave it. We'll strap it into an upcoming PR